lib/shopify_app/engine.rb

```diff
@@ -12,5 +12,9 @@ class Engine < Rails::Engine
         storage_access.svg
       ]
     end
+
+    initializer "shopify_app.middleware" do |app|
+      app.config.middleware.insert_before(ActionDispatch::Cookies, ShopifyApp::SameSiteCookieMiddleware)
```

Comment:
I'd allow people to configure this. Some apps could be using versions of Rails and rack that already does this by default, so we should not try to do it again.